package.json

```diff
@@ -47,7 +47,7 @@
     "reflect-metadata": "^0.1.10",
     "rxjs": "~6.3.3",
     "single-spa": "^4.2.0",
-    "single-spa-angular2": "^1.3.0",
+    "single-spa-angular": "^2.1.0",
```

Comment:
The simple-single-spa-webpack-example repo is not very up-to-date. I plan on updating it once single-spa-angular@3 is released.

The `single-spa-angular2` package was renamed to `single-spa-angular`, since it supports versions after angular 2. And the [2.1.0 release](https://github.com/CanopyTax/single-spa-angular/releases/tag/v2.1.0) has a bug fix for [a similar bug to the one that you described](https://github.com/CanopyTax/single-spa-angular/pull/13)